resources/metadata/fortify.yaml

```diff
@@ -0,0 +1,442 @@
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+    - name: considerSuspicious
+      type: bool
+      description: Whether suspicious issues should trigger the check to fail or not
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: true
+    - name: fprUploadEndpoint
+      aliases:
+      - name: fortifyFprUploadEndpoint
+      type: string
+      description: Fortify SSC endpoint for FPR uploads
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: /upload/resultFileUpload.html
+    - name: projectName
+      aliases:
+      - name: fortifyProjectName
+      type: string
+      description: The project used for reporting results in SSC
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: '{{list .GroupID .ArtifactID | join "-" | trimAll "-"}}'
+    - name: pythonIncludes
+      type: string
+      description: 'The includes pattern used in `scanType: ''pip''` for including
+        .py files'
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: ./**/*
+    - name: reporting
+      type: bool
+      description: Influences whether a report is generated or not
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: false
+    - name: serverUrl
+      aliases:
+      - name: fortifyServerUrl
+      type: string
+      description: Fortify SSC Url to be used for accessing the APIs
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+    - name: buildDescriptorExcludeList
+      type: string
+      description: Build descriptor files to exclude modules from being scanned
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: []
+    - name: pullRequestMessageRegexGroup
+      type: int
+      description: The group number for extracting the pull request id in `pullRequestMessageRegex`
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: 1
+    - name: deltaMinutes
+      type: int
+      description: The number of minutes for which an uploaded FPR artifact is considered to be recent and healthy, if exceeded an error will be thrown
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: 5
+    - name: spotCheckMinimum
+      type: int
+      description: The minimum number of issues that must be audited per category in the `Spot Checks of each Category` folder to avoid an error being thrown
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: 1
+    - name: fprDownloadEndpoint
+      aliases:
+      - name: fortifyFprDownloadEndpoint
+      type: string
+      description: Fortify SSC endpoint  for FPR downloads
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: /download/currentStateFprDownload.html
+    - name: projectVersion
```

Comment:
isn't `versioningModel` sufficient?
If deprecated and still required we should add this in the description?